test/support/capsule_support.rb

```diff
@@ -1,10 +1,9 @@
 module Support
   module CapsuleSupport
```

Comment:
I think this should be done in factories as traits, but I considered it out of scope here.